cost-analyzer/templates/_helpers.tpl

```diff
@@ -77,6 +77,32 @@ Network Costs name used to tie autodiscovery of metrics to daemon set pods
 {{- printf "%s-%s" .Release.Name "cluster-controller" -}}
 {{- end -}}
 
+{{- define "kubecost.kubeMetricsName" -}}
+{{- if .Values.agent }}
+{{- printf "%s-%s" .Release.Name "agent" -}}
+{{- else }}
+{{- printf "%s-%s" .Release.Name "metrics" -}}
+{{- end }}
+{{- end -}}
```

Comment:
The kubecost metrics exporter and agent are different in that the agent is a very specialized version of the exporter and adds cluster exporting and cluster info exporting.